spec/e2e/end_to_end.rb

```diff
@@ -103,20 +103,19 @@ def fetch_addresses_balances_test(wallet)
 
 def transfer_test(imported_address, new_address)
   # Transfer gwei from imported address to new address.
-  puts 'Transfering 1 Gwei from imported address to new address...'
+  puts 'Transferring 1 Gwei from imported address to new address...'
   t = imported_address.transfer(1, :gwei, new_address).wait!
   expect(t.status).to eq('complete')
   puts "Transferred 1 Gwei from #{imported_address} to #{new_address}"
 
-  # Transfer some eth for gas fee to new address.
-  t2 = imported_address.transfer(0.00001, :eth, new_address).wait!
-  expect(t2.status).to eq('complete')
-  puts "Transferred 0.00001 Eth from #{imported_address} to #{new_address}"
+  # Fund the new address with faucet.
+  faucet_tx = new_address.faucet
+  puts "Requested faucet funds: #{faucet_tx}"
 
-  # Transfer gwei back from new address to imported address.
-  t = new_address.transfer(1, :gwei, imported_address).wait!
+  # Transfer eth back from new address to imported address.
```

Comment:
was this intended?

Reply:
I think this code is in mainline https://github.com/coinbase/coinbase-sdk-ruby/blob/86f5c59d3c92bf3a03ea5fed65acdbb1518eb2a6/spec/e2e/end_to_end.rb#L115-L118

but not in 0.0.10